src/cloud-api-adaptor/ibmcloud-powervs/image/Makefile

```diff
@@ -10,8 +10,6 @@ export FORWARDER_PORT ?=
 ROOT_DIR := $(dir $(lastword $(MAKEFILE_LIST)))../../
 include $(ROOT_DIR)podvm/Makefile.inc
 
-export PATH := $(PATH):/usr/local/go/bin:$(HOME)/.cargo/bin
```

Comment:
Yep, it should be up to the user to set the correct PATH, less magic, more documentation :-)